### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by JonathanFrias**

### CODE REVIEW

**Comment by kevinchesser** [comment]
This is why all of the ladies love him

File: src/server/server.js [hunk 1]
```diff
@@ -24,25 +24,36 @@ app.post('/at/index.html', urlParser, (req, res, next) => {
...
+        var timeout = 30; //seconds
+
+        if(row.passwordAttempts > 3 && ((+new Date()) - row.lastAttempt)/1000 < timeout) {
+          var remaingingTime = parseInt(timeout - ((+new Date()) - row.lastAttempt)/1000);
+          res.write("Account '" + user + "' is locked for "+remaingingTime+" seconds.");
+          res.end();
+          return;
+        }
+        if (row.password === pass) {
+          res.redirect(302, 'game.html?a='+row.playerid);
```
Comment (JonathanFrias) [inline src/server/server.js:48]: We could use express' sessions here if we wanted to be secure.

Comment (kevinchesser) [inline src/server/server.js:48]: What exactly is res?

File: src/client/at/game/Player.js [hunk 2]
```diff
@@ -222,8 +229,26 @@ Player.prototype.registerAnimalRescued = function() {
...
+}
+
 Player.prototype.getScore = function() {
   return (this._carrotsCollected * this._carrotMultiplier) +
     (this._animalsRescued * this._animalMultiplier) +
     (this._enemiesKilled * this._enemyMultiplier);
 }
+
+Player.prototype.getStats = function() {
+  return {
```
Comment (JonathanFrias) [inline src/client/at/game/Player.js:243]: This is all the stuff that we are tracking stats on? Does anything else need to be added?

**Comment by jakewilson** [comment]
:+1:

**Pull request merged**